rewriter/Minitest.cc

```diff
@@ -108,8 +108,21 @@ class ConstantMover {
 };
 
 ast::ExpressionPtr addSigVoid(ast::ExpressionPtr expr) {
-    return ast::MK::InsSeq1(expr.loc(), ast::MK::SigVoid(expr.loc(), {}), std::move(expr));
+    core::LocOffsets declLoc;
+    if (auto *mdef = ast::cast_tree<ast::MethodDef>(expr)) {
+        declLoc = mdef->declLoc;
+    } else if (auto *cdef = ast::cast_tree<ast::MethodDef>(expr)) {
```

Comment:
Did you mean?
```suggestion
    } else if (auto *cdef = ast::cast_tree<ast::ClassDef>(expr)) {
```

Right now second condition is identical to the first one and would never be executed